src/main/java/racingcar/model/Car.java

```diff
@@ -1,15 +1,26 @@
 package racingcar.model;
 
 public class Car {
-    private int distance = 0;
+    private String name;
+    private int distance;
+
+    public Car(String name, int distance) {
+        this.name = name;
+        this.distance = distance;
+    }
 
     public int getDistance() {
         return distance;
     }
 
+    public String getName() {
+        return name;
+    }
+
     public void move(boolean isMove) {
         if(isMove) {
```

Comment:
```suggestion
        if (isMove) {
```
> 단축키(Windows : Ctrl + Alt + L. Mac : ⌥ (Option) + ⌘ (Command) + L.)를 사용한다.

단축키를 이용해서 놓친 컨벤션을 바로 잡을 수 있습니다